lstchain/io/lstcontainers.py

```diff
@@ -123,6 +123,42 @@ class DL1ParametersContainer(Container):
     trigger_time = Field(None, "trigger time")
     event_type = Field(None, "event type")
 
+    # Parameters obtained with the likelihood reconstruction method
+    lhfit_call_status = Field(-10, "Status of the processing of the event "
```

Comment:
Please don't add further fields to this already monstrous container. Make a new container for the likelihood output.